internal/units/units.go

```diff
@@ -40,6 +46,13 @@ func BytesStringBase2(b int64) string {
 	return toDecimalUnitString(float64(b), 1024.0, base2UnitPrefixes, "B")
 }
 
+func BytesString(b int64) string {
+	if bool, err := strconv.ParseBool(os.Getenv(bytesStringBase2Envar)); err == nil && bool {
```

Comment:
nit: you can ignore err check, bool can't be true if error is != nil

if v, _ := strconv.ParseBool(os.Getenv(...)); v {
  
}